src/Server/DefaultHttpServerConfiguration.php

```diff
@@ -199,6 +205,20 @@ public function getCoroutinesEnabled(): bool
             && $this->settings[self::SWOOLE_HTTP_SERVER_CONFIG_ENABLE_COROUTINE];
     }
 
+    public function getUser(): string
+    {
+        return $this->settings[self::SWOOLE_HTTP_SERVER_CONFIG_USER] ?? (extension_loaded('posix') ? posix_getpwuid(
+            posix_geteuid()
+        )['name'] : '-');
```

Comment:
Why can't the user of the script be determined without having the posix extension?